mona_sdk/client.py

```diff
@@ -1025,6 +1025,22 @@ def get_aggregated_stats_of_a_specific_segmentation(
             else get_dict_result(True, app_server_response["response_data"], None)
         )
 
+    @Decorators.refresh_token_if_needed
+    def create_openai_context_class(self, context_class, openai_endpoint=None):
+        """
+        A wrapper function for "Create new openAI context class" REST endpoint.
```

Comment:
since this is public code, can you maybe add a link to where this REST is explained? if the REST function isn't documented, at least link it to information on our openai support...